.github/workflows/cd_prepare_release.yml

```diff
@@ -0,0 +1,78 @@
+name: Prepare release
+on:
+  push:
+    branches:
+    - 'release/**'
+jobs:
+  job_compute_release_name:
+    runs-on: ubuntu-18.04
+    container:
+      image: docker://alpine/git:v2.24.3
+    outputs:
+      release_name: ${{ steps.step_compute_release_name.outputs.release_name }}
+    steps:
+      - id: step_compute_release_name
+        env:
+          GIT_REF: ${{ github.ref }}
+        run: |
+          release_name=${GIT_REF#refs/heads/release/}
+          echo "Release name=$release_name"
+          echo "::set-output name=release_name::$release_name"
+
+  job_build_test_and_generate_desc:
+    needs: job_compute_release_name
+    runs-on: ubuntu-18.04
+    container:
+      image: docker://docker:stable-git
+      options: --name=runner
+    strategy:
+      matrix:
+        android_api: [28, 29]
+        android_ndk: ["--android-ndk", ""]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build test deploy describe image
+        env:
+          RELEASE_NAME: ${{ needs.job_compute_release_name.outputs.release_name }}
+          DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+        run: ./ci_cd.sh --build --test --desc --android-api ${{ matrix.android_api }} ${{ matrix.android_ndk }}
+
+      - name: Upload image description
+        uses: actions/upload-artifact@v2-preview
+        with:
+          path: desc/output/*.txt
```

Comment:
You changed the extension to `.md`

Reply:
Yes, that is the intention of c0bb7881072997d148dae87163455b7d49f2bbf5. As those file are in MD format, and are now commited on release, this will allow to display those file through github markdown renderer.

Reply:
The generated files extension was changed but not the files uploaded. 
Are there  files which match `desc/output/*.txt` to upload ? If not, this should be replaced with desc/output/*.md or removed.

Reply:
I forgot to remove this step. Now image descriptions will also be versioned so no need to attach them with release artifact.

Reply:
Should be good